controllers/sriov-controller/dpapi-controller.go

```diff
@@ -17,6 +17,7 @@ package main
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/networkservicemesh/networkservicemesh/pkg/tools"
```

Comment:
Please, move this import into the 2nd import section where all similar imports are placed in all modified files.